helm_deploy/hmpps-manage-adjudications-api/templates/housekeeping-cronjob.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: hmppps-manage-adjudications-api-delete-orphaned
+spec:
+  schedule: "*/10 * * * *"
```

Comment:
every 10 minutes - can change it to something else - but I don't think there is any harm in doing it often.